### ISSUE TITLE
QComboBox: Icons and text are overlaped 

### ISSUE THREAD

**Issue by vallsv** [ColinDuquesnoy/QDarkStyleSheet#247]
Hi,

I am using the last version of this library from conda.

I am checking it with the application i manage.
For now i have only noticed this only one glitch, visible on the drop down list of a combobox containing actions with both text and icons.

Hope it can help.

### Description / Steps to Reproduce [if necessary]

The combobox
![Capture d’écran de 2020-11-14 14-02-08](https://user-images.githubusercontent.com/7579321/99147577-7413f100-2682-11eb-8d3d-571c0771d169.png)

When i drop down the available list
![Capture d’écran de 2020-11-14 14-01-54](https://user-images.githubusercontent.com/7579321/99147580-770ee180-2682-11eb-93f4-710980e0376a.png)

### Describe Your Environment

```
* OPERATING SYSTEM---------------------------------------------------------------
    - System........................ Linux
    - Release....................... 4.19.0-12-amd64
    - Platform...................... Linux-4.19.0-12-amd64-x86_64-with-debian-10.6
    - Version....................... #1 SMP Debian 4.19.152-1 (2020-10-18)
* PYTHON DISTRIBUTION------------------------------------------------------------
    - Version....................... 3.7.9
    - C Compiler.................... GCC 7.3.0
    - C API Version................. 1013
    - Implementation................ cpython
    - Implementation Version........ 3.7.9
* QT BINDINGS--------------------------------------------------------------------
    - PyQt5 Version................. 5.9.2
    - PyQt5 Qt Version.............. 5.9.6
* QT ABSTRACTIONS----------------------------------------------------------------
    - qtpy Version.................. 1.9.0
    - qtpy Binding.................. pyqt5
    - qtpy Binding Variable......... os.environ['QT_API']
    - qtpy Import Name.............. qtpy
    - qtpy Status................... OK
* PYTHON PACKAGES----------------------------------------------------------------
    - helpdev....................... 0.7.1
    - QDarkStyle.................... 2.8.1
```

**Comment by vallsv** [comment]
Here is few lines to reproduce with IPython
```
from PyQt5 import Qt
%gui qt
w = Qt.QComboBox()
colors = [Qt.Qt.red, Qt.Qt.green, Qt.Qt.blue]
for c in colors: 
    pixmap = Qt.QPixmap(16, 16)
    pixmap.fill(c) 
    icon = Qt.QIcon(pixmap) 
    w.addItem(icon, str(c)) 
import qdarkstyle 
qapp = Qt.QApplication().instance() 
app.setStyleSheet(qdarkstyle.load_stylesheet(qt_api="PyQt5"))
w.show()                                                     
```

**Comment by dpizetta** [comment]
Hi @vallsv, if possible try to use a more recent version of pyqt5, like 5.12 preferable because it is an LTS one, it seems they (Qt) fixed this after 5.9. Please, reopen if needed in another version or if it was impossible to use another version. Thanks for reporting and for the example.

**Comment by vallsv** [comment]
If it's already fixed, it's fine. Thanks for the check.
I can't update to an uptodate version for now. But this have to be fixed on my side, so it's fine.
Regards.
